productsPage/productsPage.css

```diff
@@ -0,0 +1,100 @@
+@import url(https://king-ui.netlify.app/styles.css);
+@import url(https://king-ui.netlify.app/Stylesheets/navbar.css);
+
+@import url(https://king-ui.netlify.app/Stylesheets/reset.css);
+
+body {
+    margin: 0;
+    display: block;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 2px solid var(--primary-color);
+    align-items: center;
+    background-color: white;
+}
+nav .text-input {
+    width: 50%;
+    padding: 0.5rem 1rem;
+}
+.parent-container {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+    width: 100%;
+}
+.sidebar-nav {
+    width: 30%;
+    height: 100%;
+    background-color: white;
+    min-width: 10rem;
+    max-width: 20rem;
+    padding: 1rem 1.5rem;
+}
+.sidebar-nav .clear-filters {
+    display: flex;
+    justify-content: space-between;
+}
+.sidebar-nav .cost-filter .title {
+    padding: 1rem 0;
+}
+.sidebar-nav .cost-filter .slider-container .slider {
+    width: 100%;
+}
+.sidebar-nav .cost-filter .slider-container .slider-values {
+    display: flex;
+    justify-content: space-between;
+}
```

Comment:
Only .slider-container .slider should work I guess

Reply:
according to me its a good practise to be specific. Correct me or enlighten me please